packages/samples/vuejs/vite.config.ts

```diff
@@ -6,7 +6,20 @@ import vueJsx from '@vitejs/plugin-vue-jsx';
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [vue(), vueJsx()],
+  plugins: [
+    vue({
+      template: {
+        compilerOptions: {
+          isCustomElement: (tag) => {
+            return (
+              tag.startsWith('atomic-') || tag === 'app-label-and-field-value'
```

Comment:
I was wondering why `app-label-and-field-value` doesn't start with `atomic`?

Reply:
So the `app-label-and-field-value` that exists in the Angular sample is there to showcase how you can wrap existing Atomic component with "your own", and how you can come up with a pattern to pass properties down to atomic (web) components.

They are not part of Atomic, and we don't necessarily need to replicate them for VueJS.

In the case of the VueJS project, what we should do is see how it looks like if someone wants to create a higher level order component using Atomic ones.

If there's really nothing special to do apart from simply passing down props exactly as is, we don't need to document anything.

For Angular, that part was particularly tricky so we wanted to document and showcase a pattern that works.

Reply:
aah alright! thanks for the explanation